server/resource_server/repository/repository.go

```diff
@@ -99,6 +100,15 @@ func (r *Repository) GetClientData(clientName string) (models.Client, error) {
 	return client, nil
 }
 
+func (r *Repository) GetClientDataByBackendURL(backendURL string) ([]models.Client, error) {
+	fmt.Println("ABABABA", backendURL)
+    var clients []models.Client
+    if err := r.connection.Where("backend_uri = ?", backendURL).First(&clients).Error; err != nil {
+        return nil, err
+    }
+    return clients, nil
+}
+
```

Comment:
can we return just one value instead of array?

Reply:
Yes, we can